op-e2e/actions/proofs/operator_fee_test.go

```diff
@@ -156,12 +155,10 @@ func Test_ProgramAction_OperatorFeeConstistency(gt *testing.T) {
 		case DepositTx:
 			aliceInitialBalance, l1FeeVaultInitialBalance, baseFeeVaultInitialBalance, sequencerFeeVaultInitialBalance, operatorFeeVaultInitialBalance = getCurrentBalances()
 
-			bobInitialBalance := balanceAt(env.Bob.Address())
-
 			// regular Deposit, in new L1 block
 			env.Alice.L1.ActResetTxOpts(t)
 			env.Alice.L2.ActSetTxToAddr(&env.Dp.Addresses.Bob)(t)
-			env.Alice.L2.ActSetTxValue(new(big.Int).SetUint64(testDepositValue))(t)
+			env.Alice.L2.ActSetGasLimit(22000)(t)
```

Comment:
```suggestion
			env.Alice.L2.ActSetGasLimit(2e6)(t)
```
let's set a high one so the bug becomes more apparent.